client/src/components/DataImportExport.vue

```diff
@@ -37,11 +37,11 @@ export default {
       this.importDialog = false;
     },
     async exportData() {
-      // this.$prompt({
-      //   title: 'Export',
-      //   text: 'Saved data to file successfully.',
-      //   positiveButton: 'Ok',
-      // });
+      await this.djangoRest.export(this.mainSession.id);
+      this.$snackbar({
+        text: 'Saved data to file successfully.',
+        timeout: 6000,
+      });
```

Comment:
Should we include some sort of feedback/loading icon? There is no visual change after pressing the export button until the snackbar is shown. Also, if an error occurs in the export process, no visual update is made.

Reply:
I'm only trying to achieve feature parity here, although that would be an excellent UX improvement. I think it would be better as a separate PR.